en/02_Developer_Guides/14_Files/05_File_Migration.md

```diff
@@ -273,23 +274,23 @@ This will take all existing files under a hash path and move them to their natur
 
 ### Do I need to re-migrate my files?
 
-No, you do not need to run the file migration task when upgrading to Silverstripe CMS 4.4 or later. 
+No, you do not need to run the file migration task when upgrading to Silverstripe CMS 4.4 or later.
 
 Your existing files will still resolve. Old hash paths will be redirected to the latest version of the file. As newer versions of existing files are published, they will be moved to their natural path.
 
 You can choose to run the migration task at any point.
 
-### What if I’m using `legacy_filenames`?
+### What if i’m using `legacy_filenames`?
```

Comment:
Capilitisation - looks like it doesn't have a regular `'` quote

Reply:
Fixed by changing to "What if I use"